### PR TITLE
[server][battlepass] Adds an endpoint to support house view in dashboard

### PR DESCRIPTION
**Pull request by saresend**
## What 
* Adds an endpoint that dumps a small *shitload* of data about houses

## Data Structure 
https://pastebin.com/xxH9TA9t - link for example output

### CODE REVIEW

File: api/live.js [hunk 1]
```diff
@@ -19,6 +19,35 @@ router.get("/tasks", async (req, res) => {
   }
 });
 
+router.get("/houseInfo", async (req, res) => {
```
Comment (jzeiders) [inline api/live.js]: We going to need some perf monitoring on this lol. Pretty much loading the entire DB

**Comment by WilhelmWillie** [comment]
@abigger87 huge implications for the hacker live dashboard

**Comment by saresend** [comment]
Works for me, will add

File: api/live.js (hunk 1 above)
Comment (saresend) [reply api/live.js]: Yeah, true, I wanna do a stress test pass on these routes once we're feature complete to make sure it can handle it

**Pull request merged**